fmriprep/cli/run.py

```diff
@@ -853,6 +635,79 @@ def build_workflow(opts, retval):
     return retval
 
 
+def parse_spaces(opts):
+    """Ensures the spaces are correctly parsed"""
+    from sys import stderr
+    from collections import OrderedDict
+    from templateflow.api import templates as get_templates
+    # Set the default template to 'MNI152NLin2009cAsym'
+    output_spaces = opts.output_spaces or OrderedDict([('MNI152NLin2009cAsym', {})])
+
+    if opts.template:
+        print("""\
+The ``--template`` option has been deprecated in version 1.4.0. Your selected template \
+"%s" will be inserted at the front of the ``--output-spaces`` argument list. Please update \
+your scripts to use ``--output-spaces``.""" % opts.template, file=stderr)
+        deprecated_tpl_arg = [(opts.template, {})]
+        # If output_spaces is not set, just replate the default - append otherwise
+        if opts.output_spaces is not None:
+            deprecated_tpl_arg += list(output_spaces.items())
+        output_spaces = OrderedDict(deprecated_tpl_arg)
+
+    if opts.output_space:
+        print("""\
+The ``--output_space`` option has been deprecated in version 1.4.0. Your selection of spaces \
+"%s" will be inserted at the front of the ``--output-spaces`` argument list. Please update \
+your scripts to use ``--output-spaces``.""" % ', '.join(opts.output_space), file=stderr)
+        missing = set(opts.output_space)
+        if 'template' in missing:
+            missing.remove('template')
+            if not opts.template:
+                missing.add('MNI152NLin2009cAsym')
+        missing = missing - set(output_spaces.keys())
+        output_spaces.update({tpl: {} for tpl in missing})
+
+    FS_SPACES = set(['fsnative', 'fsaverage', 'fsaverage6', 'fsaverage5'])
+    if opts.run_reconall and not list(FS_SPACES.intersection(output_spaces.keys())):
+        print("""\
+Although ``--fs-no-reconall`` was not set (i.e. FreeSurfer is to be run), no FreeSurfer \
+output space (valid values are: %s) was selected. Adding default "fsaverage5" to the \
+list of output spaces.""" % ', '.join(FS_SPACES), file=stderr)
+        output_spaces['fsaverage5'] = {}
+
+    # Validity of some inputs
+    # ERROR check if use_aroma was specified, but the correct template was not
+    if opts.use_aroma and 'MNI152NLin6Asym' not in output_spaces:
+        output_spaces['MNI152NLin6Asym'] = {'res': 2}
+        print("""\
+Option "--use-aroma" requires functional images to be resampled to MNI152NLin6Asym space. \
+The argument "MNI152NLin6Asym:res-2" has been automatically added to the list of output spaces \
+(option ``--output-spaces``).""", file=stderr)
+
+    if opts.cifti_output and 'MNI152NLin2009cAsym' not in output_spaces:
+        if 'MNI152NLin2009cAsym' not in output_spaces:
+            output_spaces['MNI152NLin2009cAsym'] = {'res': 2}
+            print("""Option ``--cifti-output`` requires functional images to be resampled to \
+``MNI152NLin6Asym`` space. Such template identifier has been automatically added to the list \
```

Comment:
Oh man, I thought I had changed the three instances. It should read `MNI152NLin2009cAsym`. It will change though when we support real HCP (to `MNI152NLin6Asym`). Sorry about that.